src/elements/container/container/container.ts

```diff
@@ -37,9 +43,19 @@ export class SbbContainerElement extends LitElement {
     }
   }
 
+  private _imageSlotChanged(): void {
+    const image: SbbImageElement = this.querySelector('sbb-image[slot="image"]') as SbbImageElement;
+    if (!image) {
+      return;
+    }
+
+    image.borderRadius = 'none';
```

Comment:
I think it's much easier to use the css variable --sbb-image-border-radius=0 on :slotted(sbb-image)